README.md

```diff
@@ -724,331 +690,284 @@ signIn.after('identify', async () => {
 
-var signIn = new OktaSignIn(config);
-```
+For embedded widgets, you should set the `issuer`, `clientId`, and `redirectUri`. Additionally you should set `useInteractionCodeFlow` to `true` to enable the OIE pipeline. (See [this document](https://github.com/okta/okta-signin-widget/blob/master/docs/classic.md) for details on running in "classic" pipeline).
+
+### Basic config options
+
+#### issuer
 
-### OIDC Applications
+The URL of the [Authorization Server][] which will issue [OAuth][] tokens to your application.
 
-For OIDC applications, you need to set the `clientId` and `redirectUri`. If `issuer` is not set, it will be inferred from `baseUrl`.
+Basic configuration using the "default" [Custom Authorization Server][]:
 
 ```javascript
 var config = {
-  baseUrl: 'https://{yourOktaDomain}', // issuer will be https://{yourOktaDomain}/oauth2/default
+  issuer: 'https://{yourOktaDomain}/oauth2/default',
   clientId: '{{clientId of your OIDC app}}',
-  redirectUri: '{{redirectUri configured in OIDC app}}'
+  redirectUri: '{{redirectUri configured in OIDC app}}',
+  useInteractionCodeFlow: true
 }
 ```
 
-By default, the issuer ([Authorization Server][]) will be set to the default [Custom Authorization Server][].
-
-```javascript
-// default issuer
-config.baseUrl + '/oauth2/default'
-```
-
-A different [Custom Authorization Server][] can be specified by setting the `issuer` explicitly:
+A different [Custom Authorization Server][] can be specified:
 
 ```javascript
 var config = {
   issuer: 'https://{yourOktaDomain}/oauth2/custom',
   clientId: '{{clientId of your OIDC app}}',
-  redirectUri: '{{redirectUri configured in OIDC app}}'
+  redirectUri: '{{redirectUri configured in OIDC app}}',
+  useInteractionCodeFlow: true
 }
 ```
 
-If `issuer` is set, there is no need to set `baseUrl`.
-
 Some applications, such as those that require access to the Okta User API, will want to use the Okta Organization [Authorization Server][] as the issuer. In this case the `issuer` should match your Okta domain:
 
 ```javascript
 var config = {
   issuer: 'https://{yourOktaDomain}',
   clientId: '{{clientId of your OIDC app}}',
   redirectUri: '{{redirectUri configured in OIDC app}}',
+  useInteractionCodeFlow: true
 }
 ```
 
-### Basic config options
+> **Note**: The Okta Organization [Authorization Server][] is only meant for access to the Okta User API and does not support all of the features of the standard [Custom Authorization Server][], such as custom scopes on access tokens. It is generally recommended to use a [Custom Authorization Server][] to secure access to your organization's resources.
 
-- **baseUrl:** The base URL for your Okta organization
+#### clientId
 
-    ```javascript
-    // Production org with subdomain "acme"
-    baseUrl: 'https://acme.okta.com'
+> **Note**: This configuration value can be found in the Okta Admin UI under the application's "General Settings"
```

Comment:
Since labels in our UI may change, consider linking to our [product help documentation](https://developer.okta.com/docs/guides/find-your-app-credentials/main/), which we typically ensure is updated alongside UI changes.